src/o365/spo/commands/list/list-label-set.ts

```diff
@@ -0,0 +1,241 @@
+        description: 'ID of the list where the field is located. Specify only one of listTitle, listId or listUrl'
+      },
+      {
+        option: '--listUrl [listUrl]',
+        description: 'Server- or web-relative URL of the list where the field is located. Specify only one of listTitle, listId or listUrl'
+      },
+      {
+        option: '--syncToItems [syncToItems]',
+        description: 'Specify, to set the label on all items in the list'
+      },
+      {
+        option: '--blockDelete [blockDelete]',
+        description: 'Specify, to disallow deleting items in the list'
+      },
+      {
+        option: '--blockEdit [blockEdit]',
+        description: 'Specify, to disallow editing items in the list'
+      }
+    ];
+
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+
+  public validate(): CommandValidate {
+    return (args: CommandArgs): boolean | string => {
+      if (!args.options.label) {
+        return 'Required parameter label missing';
+      }
+
+      if (!args.options.webUrl) {
+        return 'Required parameter webUrl missing';
+      }
+
+      if (!args.options.listId && !args.options.listTitle && !args.options.listUrl) {
+        return `Specify listId or listTitle or listUrl.`;
+      }
+
+      if (args.options.listId && !Utils.isValidGuid(args.options.listId)) {
+        return `${args.options.listId} is not a valid GUID`;
+      }
+
+      return SpoCommand.isValidSharePointUrl(args.options.webUrl);
+    };
+  }
+
+  public commandHelp(args: {}, log: (help: string) => void): void {
+    const chalk = vorpal.chalk;
+    log(vorpal.find(this.name).helpInformation());
+    log(
+      `  ${chalk.yellow('Important:')} before using this command, log in to a SharePoint Online site,
+    using the ${chalk.blue(commands.LOGIN)} command.
+  
+  Remarks:
+  
+    To set a list classification label, you have to first log in to SharePoint using the
+    ${chalk.blue(commands.LOGIN)} command, eg. ${chalk.grey(`${config.delimiter} ${commands.LOGIN} https://contoso.sharepoint.com`)}.
+        
+  Examples:
+  
+    Sets classification label "Confidential" for list ${chalk.grey('Shared Documents')} located in site
+    ${chalk.grey('https://contoso.sharepoint.com/sites/project-x')}
+      ${chalk.grey(config.delimiter)} ${commands.LIST_LABEL_SET} --webUrl https://contoso.sharepoint.com/sites/project-x --listUrl 'Shared Documents' --label 'Confidential'
+
+    Sets classification label "Confidential" and applies 'blockEdit','blockDelete' and 'syncToItems' flags
+    for list ${chalk.grey('Shared Documents')} located in site ${chalk.grey('https://contoso.sharepoint.com/sites/project-x')}
+      ${chalk.grey(config.delimiter)} ${commands.LIST_LABEL_SET} --webUrl https://contoso.sharepoint.com/sites/project-x --listTitle 'Documents' --label 'Confidential' --blockEdit --blockDelete --syncToItems
+
+  More information:
+
+    PnP PowerShell alternative:
```

Comment:
No need to include the link to the equivalent PowerShell cmdlet here. We do that already in the PowerShell comparison page in the docs.